tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -3480,28 +3517,24 @@ protected function basicDeleteTest(int $version): void {
   /**
    * Create and return a case object for the given Client ID.
    *
+   * Note that calling caseCreate directly is preferred.
+   *
    * @param int $clientID
    * @param int|null $loggedInUser
    *   Omit or pass NULL to use the same as clientId
    * @param array $extra
    *   Optional specific parameters such as start_date
+   * @param string $identifier
    *
    * @return CRM_Case_BAO_Case
    */
-  public function createCase(int $clientID, ?int $loggedInUser = NULL, array $extra = []): CRM_Case_DAO_Case {
-    $caseParams = array_merge([
-      'activity_subject' => 'Case Subject',
-      'client_id'        => $clientID,
-      'case_type_id'     => 1,
-      'status_id'        => 1,
-      'case_type'        => 'housing_support',
-      'subject'          => 'Case Subject',
-      'start_date'       => date('Y-m-d'),
-      'start_date_time'  => date('YmdHis'),
-      'medium_id'        => 2,
-      'activity_details' => '',
-    ], $extra);
-    return (new CRM_Case_Form_Case())->testSubmit($caseParams, 'OpenCase', $loggedInUser ?: $clientID, 'standalone');
+  public function createCase(int $clientID, ?int $loggedInUser = NULL, array $extra = [], string $identifier = 'form-created'): CRM_Case_DAO_Case {
```

Comment:
I don't see any examples in core where it's on purpose and there's obviously no tests that check the results, but this change means you can no longer mock any contact as the case creator because now $loggedInUser is ignored and it always uses whatever is currently the logged in user (see also [here](https://github.com/civicrm/civicrm-core/blob/edc1624c8d0022b95db6aebdbd38586999df632d/CRM/Case/Form/Activity/OpenCase.php#L294) and [here](https://github.com/civicrm/civicrm-core/blob/edc1624c8d0022b95db6aebdbd38586999df632d/CRM/Case/Form/Case.php#L181) and in testSubmit). There are several tests that use the client as the creator (when loggedInUser is null), but to be honest I don't think that's a realistic test - the client is rarely (never?) using the backend form - so I think I'm ok with all of those tests now using the currently logged in user.